src/containerapp/HISTORY.rst

```diff
@@ -3,6 +3,16 @@
 Release History
 ===============
 
+0.3.12
+++++++
+* Add 'az containerapp env update' to update managed environment properties
+* Add custom domains support to 'az containerapp env create' and 'az containerapp env update'
+* 'az containerapp logs show': add new parameter "--kind" to allow showing system logs
```

Comment:
should this be type instead of kind. or did we choose kind because this matches the property on the API end?

Reply:
This isn't because of the API -- I was going off of Mike Morton's design in the work item: https://msazure.visualstudio.com/Antares/_workitems/edit/13707874 

I can change it to --type